x/gov/types/v1/params.go

```diff
@@ -20,6 +20,7 @@ var (
 	DefaultThreshold              = sdk.NewDecWithPrec(5, 1)
 	DefaultVetoThreshold          = sdk.NewDecWithPrec(334, 3)
 	DefaultMinInitialDepositRatio = sdk.ZeroDec()
+	DefaultProposalCancelBurnRate = sdk.ZeroDec()
```

Comment:
Is it really a good default? The default param does not disincentivize spammy behavior. I feel like we should have good default.

Reply:
@julienrbrt  can you suggest a good default for the `proposal_cancel_burn_rate`

Reply:
As I thought it was returned to the depositor during the deposit period I would have said 50%.

Reply:
However, it seems not to be the case. We could improve the refund / burning behavior in a follow-up (being that update a rc blocker). I don't think it worth blocking this PR.

Reply:
Let's put 50% as the default.